plugins/woocommerce/includes/admin/class-wc-admin-assets.php

```diff
@@ -124,44 +150,197 @@ public function admin_styles() {
 			}
 		}
 
+		/**
+		 * Get the scripts used for registration.
+		 *
+		 * @return array
+		 */
+		private function get_scripts() {
+			$suffix  = Constants::is_true( 'SCRIPT_DEBUG' ) ? '' : '.min';
+			$version = Constants::get_constant( 'WC_VERSION' );
+
+			return array(
```

Comment:
I tried to limit this list to those dependencies that might be reusable by other plugins, though it's possible that others exist.